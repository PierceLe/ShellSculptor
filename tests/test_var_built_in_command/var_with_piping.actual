>> cd ~
>> var -s student_with_highest_mark  'cat ~/test_data/student_marks.csv | sort -t, -k2,2n | head -n 2'
cat: /Users/piercele/test_data/student_marks.csv: No such file or directory
>> echo "${student_with_highest_mark}"


