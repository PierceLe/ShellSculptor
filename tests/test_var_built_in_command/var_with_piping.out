>> cd ~
>> var -s student_with_highest_mark  'cat ~/test_data/student_marks.csv | sort -t, -k2,2n | head -n 2'
>> echo "${student_with_highest_mark}"
SID,Mark
456456456,50


